x-pack/plugins/serverless_search/public/plugin.ts

```diff
@@ -24,9 +25,10 @@ export class ServerlessSearchPlugin
   }
 
   public start(
-    _core: CoreStart,
+    core: CoreStart,
     _startDeps: ServerlessSearchPluginStartDependencies
   ): ServerlessSearchPluginStart {
+    core.chrome.project.setSideNavComponent(createComponent(core));
```

Comment:
I wonder if we should still set it through the serverless plugin as it was in the POC. not sure if there is a benefit to it

Reply:
See: https://github.com/elastic/kibana/pull/156600#discussion_r1185996989 

I think we should bring it back